pkg/install/install_test.go

```diff
@@ -179,11 +183,30 @@ var _ = Describe("Install", Label("install"), func() {
 		Expect(err.Error()).To(ContainSubstring("start failed"))
 		Expect(t.RollbackCalled()).To(BeFalse())
 	})
+	It("fails on config hook execution", func() {
+		syscall.ErrorOnChroot = true
+		err := i.Install(d)
+		Expect(err).To(HaveOccurred())
+		Expect(err.Error()).To(ContainSubstring("chroot error"))
+	})
 	It("fails on transaction commit", func() {
 		t.CommitErr = fmt.Errorf("commit failed")
 		err := i.Install(d)
 		Expect(err).To(HaveOccurred())
 		Expect(err.Error()).To(ContainSubstring("commit failed"))
 		Expect(t.RollbackCalled()).To(BeTrue())
 	})
+	It("fails on config hook execution", func() {
```

Comment:
This looks to be a duplicate. Could you please verify this?

Reply:
Indeed it is, I am removing it.